libs/components/forms/testing/src/input-box/input-box-harness.spec.ts

```diff
@@ -144,19 +144,20 @@ describe('Input box harness', () => {
     );
   });
 
-  it('should open help popover and widget when clicked', async () => {
```

Comment:
This test asserts that both popovers and help keys can be used at the same time, which isn't a pattern we want to support going forward.